### PR TITLE
Anonymous ids remap management command

### PR DESCRIPTION
**Pull request by TamoshaytisV**
## Story

https://liv-it.atlassian.net/browse/WEB-228
## Description

This management command allows to rebuild anonymous user ids if `SECRET_KEY` changes
## Test (on devstack)
1. Submit any number of problems in any section
2. Check course progress
3. Change secret key in `lms/envs/devstack.py` and restart edxapp
4. Run

```
# manage.py lms anonymousid_remap --settings=devstack
```

Restart edx. Check all submissions and progress is saved

@auraz @polesye @den99f @avkoval @leha2000 @madyasiwi

### CODE REVIEW

File: common/djangoapps/student/management/commands/anonymousid_remap.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class Command(BaseCommand):
+
+    help = """Recreate anonymized ids when django secret key was changed"""
+    can_import_settings = True
+
+    def handle(self, *args, **options):
+        from django.conf import settings
+        secret_key = settings.SECRET_KEY
+        qs = AnonymousUserId.objects.all()
+        for student in qs:
```
Comment (auraz) [inline common/djangoapps/student/management/commands/anonymousid_remap.py]: better: for anonymous_student_id

**Comment by auraz** [comment]
good job. Please print how many ids changed / from total number at the end of process.

**Comment by auraz** [comment]
@TamoshaytisV in future please use correct branch names: labster/feature/anonymousid_remap

**Comment by auraz** [comment]
:+1:

**Comment by leha2000** [comment]
I checked the algorithm against the student/models.py - looks OK for me

**Comment by polesye** [comment]
:+1:

**Pull request merged**